section9_dictionary/project/main.py

```diff
@@ -0,0 +1,34 @@
+from art import logo
+import os
+
+print(logo)
+print("Welcome to secret auction program.")
+istherebidders = True
+bidders = {}
+greatest_bid = 0
+
+
+while istherebidders:
+    
+    bidder_name = input("What is your name?: ").lower()                     
+    individual_bid = int(input("What is you bid?: $"))
+    bidders[individual_bid] = bidder_name                                               #Put each bidder and their bid into dictionaries name bidders 
+    isthereanybidders = input("Are there other bidder? Type 'yes' or 'no'.\n").lower()             #Asking user to continous the auction or not. 
+    os.system('cls')
+    
+    if isthereanybidders == "yes":
+        istherebidders = True
+    elif isthereanybidders == "no":
+        istherebidders = False                                                                      #Clear the screen of program so next bidders will not allowed to see the previous bidders name and their bid.    
+    else:
```

Comment:
good job on figuring out this case! actually, it might be better if we ask for the 'yes' or 'no' answer again instead of go ahead and assume it is 'yes'